src/components/account-detail.tsx

```diff
@@ -20,8 +24,14 @@ import {createTheme} from '@app/helpers';
 import {I18N} from '@app/i18n';
 import {Wallet} from '@app/models/wallet';
 import {sendNotification} from '@app/services';
+import {Cosmos} from '@app/services/cosmos';
 import {GRADIENT_END, GRADIENT_START} from '@app/variables/common';
 
+enum TabNames {
+  evm = 'evm',
```

Comment:
You can use getText(I18n.evmTitle) here as a value

Reply:
Type  string  is not assignable to type  number  as required for computed enum member values.

Reply:
ts bug btw: https://github.com/microsoft/TypeScript/issues/40793

Reply:
Link to issue for future investigation https://github.com/microsoft/TypeScript/issues/40793